doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,66 @@
 # 4. Kubernetes Testing Cookbook
 
 ## Table of Contents
+* [4.1 Deploy your own conformance toolchain](#4.1)
+* [4.2 Configure Kubernetes API testing](#4.2)
+* [4.3 Run Kubernetes conformance suite](#4.3)
+
+<a name="4.1"></a>
+## 4.1 Deploy your own conformance toolchain
+
+At the time of writing, the CI description file is hosted in Functest and only
+runs the containers selected by CNTT RC2. It will be completed by the
+next CNTT mandatory test cases and then a new CI description file will be
+proposed in a shared tree.
+
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) only requires
+internet access, GNU/Linux as Operating System and asks for a few
+dependencies as described in
+[Deploy your own Xtesting CI/CD toolchains](https://wiki.opnfv.org/pages/viewpage.action?pageId=32015004):
+- python-virtualenv
+- docker.io
+- git
+
+Please note the next two points depending on the GNU/Linux distributions and
+the network settings:
+- SELinux: you may have to add -\-system-site-packages when creating the
+  virtualenv ("Aborting, target uses selinux but python bindings
+  (libselinux-python) aren't installed!")
+- Proxy: you may set your proxy in env for Ansible and in systemd for Docker
+  https://docs.docker.com/config/daemon/systemd/#httphttps-proxy
+
+To deploy your own CI toolchain running CNTT Compliance:
+```bash
+virtualenv functest-kubernetes
+. functest-kubernetes/bin/activate
+pip install ansible
+ansible-galaxy install collivier.xtesting
+git clone https://gerrit.opnfv.org/gerrit/functest-kubernetes functest-kubernetes-src
+(cd functest-kubernetes-src && git checkout -b stable/kali origin/stable/kali)
+ansible-playbook functest-kubernetes-src/ansible/site.cntt.yml
+```
+
+<a name="4.2"></a>
+### 4.2 Configure Kubernetes API testing
+
+Here is the default Functest tree as proposed in
+[Run Alpine Functest containers (Kali)](https://wiki.opnfv.org/pages/viewpage.action?pageId=56295675):
+- /home/opnfv/functest-kubernetes/config (usually /etc/kubernetes/admin.conf)
```

Comment:
A suggestion to clarify this part. I also removed the link to the OPNFV wiki because I found it confusion (too much information actuatlly). However, we can keep it if considered valuable.

```suggestion
Place the kubeconfig configuration file corresponding to the Kubernetes cluster under test in the following location on the machine running the cookbook:

`/home/opnfv/functest-kubernetes/config`

```